src/mlpack/methods/ann/ffn.hpp

```diff
@@ -97,11 +97,12 @@ class FFN
    * @param predictors Input training variables.
    * @param responses Outputs results from input training variables.
    * @param optimizer Instantiated optimizer used to train the model.
+   * @return final objective value.
```

Comment:
Picky style comment, this should start with a capital letter.

Reply:
Done